### ISSUE TITLE
closeAllPanels doesn't work

### ISSUE THREAD

**Issue by ievgenkainov** [BeSite/jQuery.mmenu#456]
Good day. closeAllPanels method doesn't work because no panels are found as a direct children inside menu.
The html structure that plugin produces is 
```
<menu element>
  <div class="mm-panels">
    <div class="mm-panel"></div>
    <div class="mm-panel"></div>
 </div>
</menu element>
```

https://github.com/BeSite/jQuery.mmenu/blob/master/src/core/js/jquery.mmenu.oncanvas.js#L211
So my fix is to use .find instead of .children.

**Comment by FrDH** [comment]
Thank you for pointing out this issue, the fix would be to use `this.$pnls` instead of `this.$menu`. I'll fix this in the next version.

**Comment by FrDH** [comment]
Should be fixed in version 5.5.3
Could you check?

**Comment by ievgenkainov** [comment]
Checked. 
btw, you can reuse the '''$pnls''' you already got above.
https://github.com/BeSite/jQuery.mmenu/commit/1d2a985c2c7de56bcfb716221108bad1b1e0d142#diff-49a5625a0b73da15da28852aeecf965aL215

Functionality works, but the animation is a little weird. Because 'hidden' class applied to the opened panel right away, so there is animation only for first panel.

But what i did, i simply call '''this.openPanel( $frst );''' without this piece of code before
'''
this.$pnls.children( '.' + _c.panel )
    .not( $frst )
    .removeClass( _c.subopened )
    .removeClass( _c.opened )
    .removeClass( _c.current )
    .removeClass( _c.highest )
    .addClass( _c.hidden );
'''

So in this case 'this.openPanel' does all job and animation applies correctly.
